LEGO1/mxaudiopresenter.h

```diff
@@ -5,9 +5,10 @@
 #include "mxmediapresenter.h"
 
 // VTABLE 0x100d4c70
+// SIZE 0x54
 class MxAudioPresenter : public MxMediaPresenter {
 public:
-	MxAudioPresenter() { m_unk50 = 100; }
+	MxAudioPresenter() { m_volume = 100; }
```

Comment:
do we know where this constructor is or is it just a guess?

Reply:
It's where generally any other inlined function are as well, in the function bodies that call them. In this case, examples include `LegoLoadCacheSoundPresenter`, `MxMusicPresenter` and `MxMIDIPresenter`, all of which match 100%.